docking/relative_docking.py

```diff
@@ -0,0 +1,326 @@
+"""
+1. Solvates a protein, minimizes w.r.t guest_A, equilibrates & spins off switching jobs
+   (deleting guest_A while inserting guest_B), calculates work
+2. Does the same thing in solvent instead of protein
+3. Repeats 1 & 2 for the opposite direction (guest_B --> guest_A)
+"""
+import os
+import time
+import numpy as np
+
+from md import builders, minimizer
+from fe import free_energy, topology
+from ff import Forcefield
+from ff.handlers.deserialize import deserialize_handlers
+from timemachine.lib import custom_ops, LangevinIntegrator
+
+from testsystems.relative import hif2a_ligand_pair
+
+from docking import report
+
+MAX_LAMBDA = 1.0
+MIN_LAMBDA = 0.0
+TRANSITION_STEPS = 501
```

Comment:
An interesting question is how many transition steps we need to still get reasonable enrichment (even though the estimates themselves may be quite poor)